File: Pemro1_Modul4/Python/PRAK405-2510817320010-PutriDeissytaHerliyana.py
n, kelipatan = map(int, input().split())

total_semua = 0

for i in range(1, n + 1):
    total_baris = 0

    for j in range(i, 0, -1):
        print(f"({j} * {kelipatan})", end="")
        total_baris += j * kelipatan
        if j != 1:
            print(" + ", end="")
    print(f" = {total_baris}")
    total_semua += total_baris

print(total_semua)

File: Pemro1_Modul4/C/PRAK405-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);

    int total_semua = 0;

    for (int i = 1; i <= n; i++) {
        int total_baris = 0;

        for (int j = i; j >= 1; j--) {
            printf("(%d * %d)", j, kelipatan);
            total_baris += j * kelipatan;
            if (j != 1) {
                printf(" + ");
            }
        }

        printf(" = %d\n", total_baris);
        total_semua += total_baris;
    }

    printf("%d\n", total_semua);

    return 0;
}
